daemon/volumes.go

```diff
@@ -250,6 +250,9 @@ func (daemon *Daemon) verifyOldVolumesInfo(container *Container) error {
 	}
 
 	for destination, hostPath := range vols.Volumes {
+		if container.MountPoints == nil {
```

Comment:
Any reason not to move this check into addLocalMountPoint() to catch all possible calls to it (in the future too)?
addMountPointWithVolume() too, I guess.